lib/redux/client/client_actions.dart

```diff
@@ -290,16 +292,66 @@ void handleClientAction(
           context: context));
       break;
     case EntityAction.restore:
-      store.dispatch(RestoreClientRequest(
-          snackBarCompleter(context, localization.restoredClient), client.id));
+      if (multiselect && !isMultiselectLast) {
```

Comment:
Rather than have an if/else here I think it would be better to simply pass all of the ids to the the RestoreClientRequest action, same for the other actions